core-services/prow/02_config/_plugins.yaml

```diff
@@ -369,25 +369,25 @@ config_updater:
         - ci
         - ci-stg
       name: ci-operator-misc-configs
-    ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-feature*.yaml:
+    ci-operator/config/**/cluster-logging-operator/openshift-cluster-logging-operator-feature*.yaml:
       clusters:
         default:
         - ci
         - ci-stg
       name: ci-operator-misc-configs
-    ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-feature*.yaml:
+    ci-operator/config/**/elasticsearch-operator/openshift-elasticsearch-operator-feature*.yaml:
       clusters:
         default:
         - ci
         - ci-stg
       name: ci-operator-misc-configs
-    ci-operator/config/openshift/kubernetes/*.yaml:
+    ci-operator/config/**/origin-aggregated-logging/openshift-origin-aggregated-logging-feature*.yaml:
       clusters:
         default:
         - ci
         - ci-stg
       name: ci-operator-misc-configs
-    ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-feature*.yaml:
+    ci-operator/config/openshift/kubernetes/*.yaml:
```

Comment:
We need to do the same also for the files under `ci-operator/jobs`

Reply:
addressed